can_remote/client.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, channel, ssl_context=None, **config):
             ssl_context = DEFAULT_SSL_CONTEXT
         websocket = WebSocket(url, ["can.binary+json.v1", "can.json.v1"],
                               ssl_context=ssl_context)
-        self.protocol = RemoteClientProtocol(config, websocket)
+        self._can_protocol = RemoteClientProtocol(config, websocket)
```

Comment:
I believe this attribute has a different meaning in Python-can. It should probably be named something else that is not in use.